src/flex/FlexView.js

```diff
@@ -43,6 +43,7 @@ export default React.createClass({
       React.PropTypes.number,
       React.PropTypes.string
     ]),
+    events: React.PropTypes.object,
```

Comment:
not sure about `events` name: this is just an object of additional props, so maybe `props` would be more appropriate?